experiments/harmless/finetune.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import json
+import torch
+import argparse
+import autocrit
+from datasets import load_dataset
+from accelerate import Accelerator
+from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline
+from generate import evaluate_unsafe
+
+if __name__ == '__main__':
```

Comment:
Again do you think it makes sense to have this training script here or in trlX? In my mind autocrit is primarily for llm inference and trlX is primarily for training (since we already have SFT code over there anyway)